resources/kubernetes/operator/helm/values.yaml

```diff
@@ -121,8 +121,8 @@ collectors:
               - "k8s.pod.uid"
               - "k8s.pod.start_time"
             labels:
-              - tag_name: app.label.component
-                key: app.kubernetes.io/component
```

Comment:
@ChrsMark for some reason here was used `app.kubernetes.io/component` for `service.name` - it was introduced in the original version of the manifest, maybe you remember reasoning for that? or it was a mistake and it should be `app.kubernetes.io/name` ?

Reply:
If I remember correctly it was an assumption borrowed from OTel demo: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/3a6fff3f065dafcdcf7041885c5d0d1bf61a1167/charts/opentelemetry-demo/values.yaml#L4-L14.

I'm not sure if that's coming from a spec definition or something so I think it's not a hard rule. However, whatever we choose we should ensure that it's the way forward. See https://github.com/open-telemetry/semantic-conventions/issues/236

@rogercoll do you have maybe more context on this?

Reply:
thanks!
by those changes I am replicating ingest pipeline we use for logs - https://github.com/elastic/integrations/blob/main/packages/kubernetes/data_stream/container_logs/elasticsearch/ingest_pipeline/default.yml 

and it seems to align with the candidates in https://github.com/open-telemetry/semantic-conventions/issues/236

Reply:
My only concern is that OTel demo uses the `app.kubernetes.io/component` which might bring issues in signal correlation. I wonder if we could propose this as a change to OTel demo.

Reply:
Can we have both labels?

Reply:
> My only concern is that OTel demo uses the app.kubernetes.io/component which might bring issues in signal correlation. I wonder if we could propose this as a change to OTel demo.

Good catch! I agree with you both, the OTel demo should be using `app.kubernetes.io/name` instead of the current `app.kubernetes.io/component`. At least based on the definition of this well-known attributes: https://kubernetes.io/docs/reference/labels-annotations-taints/ and SemConv.

Just created this issue upstream to track the changes: https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1383